packages-next/fields/src/types/timestamp/views/index.tsx

```diff
@@ -79,25 +134,33 @@ export const CardValue: CardValueComponent = ({ item, field }) => {
   );
 };
 
-function isValid(value: string) {
-  if (value === '') return true;
-  try {
-    return parseISO(value).toISOString() === value;
-  } catch (err) {
-    return false;
-  }
-}
-
-export const controller = (config: FieldControllerConfig): FieldController<string, string> => {
+export const controller = (
+  config: FieldControllerConfig
+): FieldController<{ dateValue: string; timeValue: string }, string> => {
   return {
     path: config.path,
     label: config.label,
     graphqlSelection: config.path,
-    defaultValue: '',
-    deserialize: data => data[config.path] || '',
-    serialize: value => ({ [config.path]: value === '' ? null : value }),
-    validate(value) {
-      return isValid(value);
+    defaultValue: { dateValue: '', timeValue: '' },
+    deserialize: data => {
+      const value = data[config.path];
+      if (value) {
+        return deconstructTimestamp(value);
+      }
+      return { dateValue: '', timeValue: '' };
+    },
+    serialize: ({ dateValue, timeValue }) => {
```

Comment:
important note here is that due to the fact that we're constructing a value in `serialize`, and `serialize` is called on every form update in the `ItemPage` component, this causes a peculiar case where, specifying a time value without a date value will not cause the save changes button to be enabled. 

Ultimately i think this is harmless, as the user generally shouldn't be able to create a timestamp without a date value anyways, but it is a UX flow that isn't necessarily consistent with field behaviour. 

We can potentially mitigate this issue by having the time field conditionally appear *after* the date value has been selected.